cdap-ui/app/cdap/components/PipelineScheduler/ProfilesForSchedule/ProfilesForSchedule.scss

```diff
@@ -22,9 +22,13 @@
       display: flex;
       justify-content: space-between;
       min-width: 150px;
+      max-width: 400px;
```

Comment:
nit: may be should have it in a variable. We have some numbers we use for width.